Source/FakeItEasy/Core/FakeWrapperConfigurator.cs

```diff
@@ -47,6 +47,13 @@ public IFakeOptionsBuilder<T> WithAdditionalAttributes(IEnumerable<CustomAttribu
 
         public IFakeOptionsBuilder<T> Implements(Type interfaceType)
         {
+            Guard.AgainstNull(interfaceType, "interfaceType");
+
+            if (!interfaceType.IsInterface)
+            {
+                throw new ArgumentException("The specified type is not an interface");

```

Comment:
Yep I can reword that, I literally took the comments from the existing XML exceptions on IFakeOptionsBuilder.Implements(), would you like me to update that also?

According to IFakeOptionsBuilder.Implements<TInterface>(), that implementation should also throw for non-interfaces.  I don't have the code to hand, but do you want the same fix applied to that (if required)?